website/views/shared/profiles/achievements.jade

```diff
@@ -135,6 +139,13 @@ div(ng-if=':: profile.achievements.originalUser || profile.achievements.veteran'
     small!=env.t('originalUserText')
```

Comment:
You're very close to being done. Looks like you still have text related to the original user achievement in the veteran achievement. You can see the very end of it on this line. Remove lines 137-139 and you should be good to go. For reference, here's the whole code block with comments about which lines should be removed:

``` jade
div(ng-if=':: profile.achievements.veteran')
  .achievement.achievement-cake
  div(ng-if='::profile.achievements.veteran')
    h5=env.t('veteran')
    small=env.t('veteranText')
  div(ng-if='::profile.achievements.originalUser') // Remove me
    h5=env.t('originalUser') // Remove me
    small!=env.t('originalUserText') // Remove me
  hr

div(ng-if=':: profile.achievements.originalUser')
  .achievement.achievement-alpha
  div(ng-if='::profile.achievements.originalUser')
    h5=env.t('originalUser')
    small!=env.t('originalUserText')
  hr
````